test/integration/base/test_render.py

```diff
@@ -23,11 +23,11 @@ def click_redo(self):
 
 
 def test_jltr001r_undo_redo(dashjl):
-    fp = jl_test_file_path("jltr001r_undo_redo.jl") 
+    fp = jl_test_file_path("jltr001r_undo_redo.jl")
     dashjl.start_server(fp)
     dashjl.wait_for_element_by_css_selector(
-        "#a"
-    )   
+        "#a", timeout=4
```

Comment:
All these extra timeouts are awkward - we already [wait for the server to report it's accessible](https://github.com/plotly/dash/blob/fed4a9123f5d0cf6213da157bb94f5e1900c3440/dash/testing/application_runners.py#L433) and then [wait for the page to have some dash content](https://github.com/plotly/dash/blob/fed4a9123f5d0cf6213da157bb94f5e1900c3440/dash/testing/browser.py#L294) - is there something else we need to wait for after those two?

Not an issue to be solved in this repo, the fix needs to be in `dash.testing`, just curious if you have any insight about what's going on.

Reply:
@alexcjohnson In fact, here we are waiting for the first callback (i.e. `_dash_update_component`) to be executed. Since the Julia is JIT compiler, the corresponding code is compiled at this moment. After the release, I plan to do optimization on this side - see where I can more accurately define the types and add the ability to precompile when installing the package.

Reply:
OK cool - I suspect it'll be relatively important to get this right for the developer experience, but fine to omit for now. Is there an issue for this in which we can mention the goal to remove all these extra test timeouts?